angularjs-portal-home/src/main/webapp/partials/marketplace-portlet.html

```diff
@@ -83,10 +83,13 @@
     </div>
     <div class="col-xs-12 col-sm-9 portlet-screenshots" ng-if="portlet.marketplaceScreenshots.length !== 0">
       <h3>Screenshots</h3>
-      <ul >
-        <li ng-repeat="screenshot in portlet.marketplaceScreenshots">
-          <img ng-src="{{ ::screenshot.url }}" alt="{{ ::screenshot.captions }}">
-          <p ng-repeat="caption in screenshot.captions">{{ ::caption }}</p>
+      <ul class="enlarge" ng-repeat="screenshot in portlet.marketplaceScreenshots">
```

Comment:
Probably want to put the ng-repeat on the li instead of the ul to be more semantic.